pkg/apis/configuration/validation/transportserver_test.go

```diff
@@ -439,6 +439,27 @@ func TestValidateSessionParameters(t *testing.T) {
 	}
 }
 
+func TestValidateSessionParametersFails(t *testing.T) {
+	tests := []struct {
+		parameters *v1alpha1.SessionParameters
+		msg        string
+	}{
+		{
+			parameters: &v1alpha1.SessionParameters{
+				Timeout: "-1s",
+			},
+			msg: "[sessionParameters.timeout: Invalid value: \"-1s\": Invalid time string]",
```

Comment:
for consistency with the existing tests, where we put a short description of a case, not the string value of the error.
```suggestion
			msg: "invalid timeout",
```